lumicks/pylake/nb_widgets/tests/test_fd_selector.py

```diff
@@ -93,7 +93,7 @@ def test_selector_widget(mockevent):
     event = mockevent(selector._axes, 900, 1, rmb, False)
     selector.handle_button_event(event)
     assert selector.current_range == []
-    np.testing.assert_allclose(selector.ranges, [[]])
```

Comment:
> testing: empty range list is just an empty list

Computer science would disagree with this: The set that only contains the empty set is still a (super) set with cardinality of one :)

Reply:
ghehe, yes :D